src/instructlab/configuration.py

```diff
@@ -786,125 +682,10 @@ class Config(BaseModel):
-                "lora_quantize_dtype": None,
-                "lora_rank": 4,
-                "nproc_per_node": 1,
-                "deepspeed_cpu_offload_optimizer": False,
-                "additional_args": {},
-                "is_padding_free": False,
-            },
-            "version": CONFIG_VERSION,
-        }
-
-        for key, val in defaults.items():
-            if values.get(key) is None and val is not None:
-                values[key] = val
-
-        return values
-
-
-def finish_cfg_section(
-    defaults: dict, values: typing.Dict[str, typing.Any]
-) -> typing.Dict[str, typing.Any]:
-    for key, val in defaults.items():
-        if values.get(key) is None and val is not None:
-            values[key] = val
-
-    return values
-
 
 def get_default_config() -> Config:
     """Generates default configuration for CLI"""
-    return Config(
-        version=CONFIG_VERSION,
-        chat=_chat(
-            model=DEFAULTS.DEFAULT_MODEL,
-        ),
-        generate=_generate(
-            model=DEFAULTS.DEFAULT_MODEL,
-            taxonomy_path=DEFAULTS.TAXONOMY_DIR,
-            taxonomy_base=DEFAULTS.TAXONOMY_BASE,
-        ),
-        serve=_serve(
-            model_path=DEFAULTS.DEFAULT_MODEL,
-            llama_cpp=_serve_llama_cpp(
-                gpu_layers=-1,
-                max_ctx_size=4096,
-                llm_family="",
-            ),
-            vllm=_serve_vllm(
-                llm_family="",
-                max_startup_attempts=120,
-            ),
-        ),
-        train=_train(
-            model_path=DEFAULTS.MODEL_REPO,
-            data_path=DEFAULTS.DATASETS_DIR,
-            ckpt_output_dir=DEFAULTS.CHECKPOINTS_DIR,
-            data_output_dir=DEFAULTS.INTERNAL_DIR,
-            max_seq_len=4096,
-            max_batch_len=10000,
-            num_epochs=10,
-            effective_batch_size=3840,
-            save_samples=250000,
-            lora_quantize_dtype="nf4",
-            lora_rank=4,
-            nproc_per_node=1,
-            deepspeed_cpu_offload_optimizer=False,
-            additional_args={},
-            is_padding_free=False,
-            phased_phase1_num_epochs=10,
-            phased_phase1_effective_batch_size=128,
-            phased_phase2_num_epochs=10,
-            phased_phase2_effective_batch_size=3840,
-            phased_mt_bench_judge=DEFAULTS.DEFAULT_JUDGE_MODEL,
-        ),
-        evaluate=_evaluate(
-            base_model=DEFAULTS.MODEL_REPO,
-            mt_bench=_mtbench(
-                judge_model=DEFAULTS.DEFAULT_JUDGE_MODEL,
-                output_dir=DEFAULTS.EVAL_DATA_DIR,
-                max_workers=16,
-            ),
-            mmlu=_mmlu(
-                few_shots=5,
-                batch_size="auto",
-            ),
-            mt_bench_branch=_mtbenchbranch(taxonomy_path=DEFAULTS.TAXONOMY_DIR),
-            mmlu_branch=_mmlubranch(tasks_dir=DEFAULTS.DATASETS_DIR),
-        ),
-    )
+    return Config()
```

Comment:
at this point the necessity of this function is questionable, but it can be cleaned up later if someone wants to follow up.

Reply:
Yeah, thought about that too but it's called in a bunch of places, and was trying to minimize the changes (or lazy?) :). Thanks!